### PR TITLE
EWL-7884: Forums listing element in SG2

### PR DESCRIPTION
**Pull request by plibby**
## Ticket(s)

**Github Issue**
- N/A

**Jira Ticket**
- [EWL-7884: Forums listing element in SG2](https://issues.ama-assn.org/browse/EWL-7884)

## Description
Created new element for forums listing element.

## To Test
- [ ] Pull branch
- [ ] Run gulp serve
- [ ] Verify the following elements match the zeplin:
   http://localhost:3000/?p=molecules-forum-listing

Zeplin:
https://app.zeplin.io/project/5e56f2d8a3e66965c0092ba0/screen/5e56f3228d27f0666e6e5235

## Visual Regressions

_Please provide the pull request ID below. This will then reference the automated VRT report after it has been generated._

A report is available [here](http://htmlpreview.github.io/?https://github.com/AmericanMedicalAssociation/ama-style-guide-2/blob/visual-regression-testing-artifact/[PULL_REQUEST_ID]/html_report/index.html).

**Before proceeding:** Run visual regression tests locally to ensure new work does not introduce new bugs.

_The visual regression tests always compare against the production version of the style guide. Changes and new work will show up as test failures. Please describe your work so that the report can be reviewed by the team._

If you are creating or updating a pattern be sure you have created or updated the [scenario in `backstop.json`](ama-style-guide-2/styleguide/backstop.json).

_After changes in the report are documented and demonstrate your desired changes, please mark the pull request as `ready for review`.

For more information on visual regression testing review the [How do I run tests?](https://issues.ama-assn.org:8446/confluence/pages/viewpage.action?pageId=23298568) document in Confluence.

## Relevant Screenshots/GIFs
N/A

## Remaining Tasks
N/A

## Additional Notes
N/A

---

[Guidelines for Contribution](CONTRIBUTING.md)
[SG2 Standards](ama-style-guide-2/docs/standards.md)

### CODE REVIEW

File: styleguide/source/assets/scss/02-molecules/_forum-list.scss [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+        font-weight: normal;
+      }
+    }
+
+    tr {
+      border: none;
+      border-top: 1px solid #cecece;
+
+      .forum-title {
+        font-size: 22px;
```
Comment (caharder) [inline styleguide/source/assets/scss/02-molecules/_forum-list.scss:25]: Add font-weight to forum title
a {
  font-weight: 600;
}

File: styleguide/source/assets/scss/02-molecules/_forum-list.scss [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+      }
+
+      .subforum-listing {
+        margin-left: 2em;
```
Comment (caharder) [inline styleguide/source/assets/scss/02-molecules/_forum-list.scss:29]: Add font-weight to forum listings
a {
  font-weight: 600;
}

**Review by caharder — CHANGES_REQUESTED** [review]
Center the topics and posts, but not last post
Pattern_Lab_-_molecules-forum-listing

Add space here:
Pattern_Lab_-_molecules-forum-listing

Also, remove the header, we will need to build that into an organism or template.

**Comment by plibby** [comment]
> Center the topics and posts, but not last post
> Pattern_Lab_-_molecules-forum-listing
> 
> Add space here:
> Pattern_Lab_-_molecules-forum-listing
> 
> Also, remove the header, we will need to build that into an organism or template.

Addressed.

**Review by caharder — CHANGES_REQUESTED** [review]
One more thing. We do need the Posts and Topics centered in this.

Can you add something like this?
`.ama__forum-table-wrap table tr [data-th="Topics"], .ama__forum-table-wrap table tr [data-th="Posts"] {
    text-align: center;
}`

**Comment by plibby** [comment]
> One more thing. We do need the Posts and Topics centered in this.
> 
> Can you add something like this?
> `.ama__forum-table-wrap table tr [data-th="Topics"], .ama__forum-table-wrap table tr [data-th="Posts"] { text-align: center; }`

This is included now. I had made these changes in the feature/EWL-7868-Create-template-for-the-forum-page-in-SG2 branch so I just merged that here.

**Review by caharder — APPROVED** [review]
Great!

**Pull request merged**